src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -335,12 +333,12 @@
 
   "entity.worker.inventoryfullchestfull": "Please make space in my chests/racks. I am not able to dump my inventory.",
 
-  "entity.builder.messagebuildstart": " started building %s.",
-  "entity.builder.messagebuildcomplete": " finished building %s.",
-  "entity.builder.messagebuildcomplete_manual": " finished building %s. You can assign them another task now!",
-  "entity.builder.messagebuildcomplete_generic": "%s finished building %s.",
-  "entity.builder.messagebuildcomplete_generic_manual": "%s finished building %s. You can assign them another task now!",
-  "entity.builder.messageremovalcomplete": " finished deconstructing building %s. §6 To move the building while retaining its current level, use the 'Pick Up' button on the building's 'Build Options' GUI.",
+  "entity.builder.messagebuildstart": ": I've started work on %s!",
```

Comment:
I've started working on?

Reply:
no, here `work` is being used as a noun and not a verb

Reply:
mmm, raven's grammar is correct here